userguide/tutorials/metrics-cloudwatch.adoc

```diff
@@ -0,0 +1,157 @@
+
+```
+curl https://s3.dualstack.us-east-1.amazonaws.com/aws-xray-assets.us-east-1/xray-daemon/aws-xray-daemon-3.x.deb -o /home/ubuntu/xray.deb
+sudo dpkg -i /home/ubuntu/xray.deb
+```
+
+If you have more than one instance, repeat these commands on each one.
+
+== Create an IAM Policy
+
+This step creates a policy, or a set of permissions, that is required to access the Kill Bill metrics.
+
+. From the EC2 Dashboard, Type "IAM" in the search field at the upper left. Select the IAM Dashboard.
+. From the left menu, choose Policies, then select *Create Policy*.
+. On the page that appears, select the JSON tab. You should see:
++
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/create-iam-policy.png[align=center]
+
+
+. Next, copy the following lines between the square brackets:
++
+```
+        {
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeTags",
+                "cloudwatch:PutMetricData",
+                "xray:GetSamplingRules",
+                "xray:GetSamplingTargets",
+                "xray:GetSamplingStatisticSummaries",
+                "xray:PutTraceSegments",
+                "xray:PutTelemetryRecords",
+                "logs:CreateLogGroup",
+                "logs:CreateLogStream",
+                "logs:DescribeLogGroups",
+                "logs:DescribeLogStreams",
+                "logs:PutLogEvents"
+            ],
+            "Resource": "*"
+        }
+```
+
+. Choose *Next:Tags*.  Then on the next page, click *Next:Review*.
+
+. Now give your policy a name, such as "CloudWatch_Policy". Then click *Create Policy*.
+
+Your policy will be created.
+
+== Create an IAM Role
+
+A role is an identity that has specific permissions defined by a policy. A role can be used to define the permissions for an EC2 instance.
+
+. Return to the IAM Dashboard.
+. From the left menu, choose Roles, then select *Create Role*. You should see:
++
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/create-iam-role.png[align=center]
+
+. Under *Common Use Cases*, select *EC2*. Then click *Next*.
+. On the next page, select the policy you created. Then click *Next*.
+. Give your role a name, such as "CloudWatch_Role". Then click *Create Role*.
+
+Your role will be created.
+
+
+
+== Attach Role to EC2 Instance
+
+The last step is to associate your role with your instance(s).
+
+. Go to your EC2 Dashboard and select *Instances*.
+
+. Select your instance. If you have more than one select only one.
+
+. Select *Actions/Security/Modify IAM Role*.
+
+. From the dropdown list choose the role you just created.
+
+. Click *Save*.
+
+If you have multiple instances, repeat these steps for each one. You may use the same role for all your instances.
+
+== Test the Installation
+
+To test your installation, Go to the Cloudwatch console. After a short delay, you
+should see a set of available metrics. There should be well over 1,000 in your list.
```

Comment:
Do we need to tell users how to "**Go to the Cloudwatch console**" or is this something very obvious and we can assume users are aware of it?

Reply:
I would think they can google it up if this is not obvious.